README.md

```diff
@@ -841,23 +841,26 @@ Kafka top-level configuration contains generic options that fit both publisher a
+|----------------|---------------------------------------------------------------------------------------------------------------------------------|-------------------|----------------------------------------------|--------|-----------------|
+| Max attempts   | Maximum amount of times to retry sending a failing Message                                                                      | `max_attempts`    | `APP_PUBSUB_KAFKA_PUBLISHER_MAX_ATTEMPTS`    | int    | 3               |
+| Write timeout  | Local message timeout. This value is only enforced locally and limits the time a produced message waits for successful delivery | `write_timeout`   | `APP_PUBSUB_KAFKA_PUBLISHER_WRITE_TIMEOUT`   | string | 10s             |
+| Topic          | Topic name                                                                                                                      | `topic`           | `APP_PUBSUB_KAFKA_PUBLISHER_TOPIC`           | string | topic           |
+| Enable Metrics | Enable publishing metrics for Kafka producer                                                                                    | `metrics_enabled` | `APP_PUBSUB_KAFKA_PUBLISHER_METRICS_ENABLED` | bool   | true, false     |
 
 **Subscriber**:
 
-| Setting  | Description                                                                            | YAML variable | Environment variable (ENV)             | Type   | Possible Values |
-|----------|----------------------------------------------------------------------------------------|---------------|----------------------------------------|--------|-----------------|
-| Topic    | Topic name                                                                             | `topic`       | `APP_PUBSUB_KAFKA_SUBSCRIBER_TOPIC`    | string | topic           |
-| Group ID | Client group id string. All clients sharing the same group.id belong to the same group | `group_id`    | `APP_PUBSUB_KAFKA_SUBSCRIBER_GROUP_ID` | string | service-name    |
+| Setting        | Description                                                                            | YAML variable     | Environment variable (ENV)                    | Type   | Possible Values |
+|----------------|----------------------------------------------------------------------------------------|-------------------|-----------------------------------------------|--------|-----------------|
+| Topic          | Topic name                                                                             | `topic`           | `APP_PUBSUB_KAFKA_SUBSCRIBER_TOPIC`           | string | topic           |
+| Group ID       | Client group id string. All clients sharing the same group.id belong to the same group | `group_id`        | `APP_PUBSUB_KAFKA_SUBSCRIBER_GROUP_ID`        | string | service-name    |
```

Comment:
> All clients sharing the same group.id belong to the same group

I'm not sure I understand this sentence. Did you mean `All clients are sharing the same group. Id belongs to...`?

Reply:
@terranisu I it taken from [the librdkafka documentation](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md)

Reply:
oh, I see now. The dot in `group.id` is what confused me. Can we remove it then? In the official documentation, the dot is part of the name of the parameter `group.id` so it's clear why we have `group.id` in the description. In our case, we don't need that dot, right?

Reply:
@terranisu I have removed `.`